bcda/models/models.go

```diff
@@ -323,15 +323,26 @@ func GetSuppressedBlueButtonIDs(db *gorm.DB) []string {
 
 	var suppressedBBIDs []string
 
-	db.Raw(`SELECT DISTINCT s.blue_button_id
-			FROM (
-				SELECT blue_button_id, MAX(effective_date) max_date
-				FROM suppressions
-				WHERE effective_date <= NOW() AND preference_indicator != '' AND blue_button_id != '' AND blue_button_id IS NOT NULL
-				GROUP BY blue_button_id
-			) h
-			JOIN suppressions s ON s.blue_button_id = h.blue_button_id and s.effective_date = h.max_date
-			WHERE preference_indicator = 'N'`).Pluck("blue_button_id", &suppressedBBIDs)
+	// Set the window that the suppression query should examine. Will default to 60 if no value is provided
+	var suppressionLookback string
+	if os.Getenv("BCDA_SUPPRESSION_LOOKBACK") == "" {
```

Comment:
Can set a default int using a helper/utility function that we have...
For example: https://github.com/CMSgov/bcda-app/blob/0bb56d709e7557409630258a85f801fcbba3f635/bcda/client/bluebutton.go#L158